CONTRIBUTING.md

```diff
@@ -42,11 +41,12 @@ This contribution policy will evolve over time. For now it is based on a mixture
 #### Pull Request Requirements (:exclamation:)
 
 1. A PR **SHOULD** be a minimal and accurate answer to exactly one identified and agreed problem.
-2. A PR **SHALL NOT** include non-trivial code from other projects unless the Contributor is the original author of that code.
-3. A PR **MUST** pass all tests on at least the principle target platform.
-4. A PR **MUST** include new tests for any new functionality introduced.
-5. A PR **SHOULD** avoid "callback-hell" style and instead prefer "async/await" style.
-6. A PR **MUST** follow the requirements spelled out in this project's [Style Guide](docs/Style-Guide.md).
+2. A PR **SHOULD** follow the scout rule: leave the code cleaner than you found it.
```

Comment:
I would clarify it better.

```
2. A PR **SHOULD** follow the scout rule: leave the code cleaner than you found it when the refactor effort is not to big or out of scope from the PR.
```

Reply:
How did it get shortened to just "scout rule"? I liked "boy scout rule", and maybe since others have used it, it will become a term of art?

Or, if we're concerned about pronouns, maybe this can read "boy/girl scout rule", just so that it's clear where it comes from?

Also, whatever we agree to call it, can you please also linkify it to a more detailed explanation? I think linking it to the discussion here would work just fine: https://github.com/okTurtles/group-income-simple/issues/383#issuecomment-383381863

Reply:
I come from a language that doesn't have gendered pronouns or anything gendered really, so for me it's always strange to see that in English, and to some level I welcome the current trend of avoiding genders whenever possible. But if a scout doesn't mean the same as a boy/girl scout for you, I will go back to boy scout - for me 'boy/girl' only says that we are trying too hard to be gender neutral, and failing :)